cc/scheduler/scheduler.cc

```diff
@@ -364,6 +364,7 @@ void Scheduler::PostPendingBeginFrameTask() {
 
   if (is_idle && needs_begin_frames && has_pending_begin_frame_args &&
       has_no_pending_begin_frame_task) {
+    recordreplay::Assert("[RUN-1230] Scheduler::PostPendingBeginFrameTask #1");
```

Comment:
hi @bhackett1024 and @klochek, as a reminder:
To better help us differentiate between different stages of the crash fix process, we now use a `RUN-X-Y` label. This should have been `[RUN-1230-1454]`.